docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md

```diff
@@ -0,0 +1,211 @@
+---
+author:
+  name: Andrew Leap
+  email: andyleap@gmail.com
+description: 'Alpine Linux is a small, security oriented linux distro.  This guide details installing and configuring Alpine Linux on a KVM Linode'
+keywords: 'alpine,custom,finnix,lish,kvm'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Thursday, March 31st, 2016'
+modified: Thursday, March 31st, 2016
+modified_by:
+  name: Andrew Leap
+title: 'Install Alpine Linux on your Linode'
+contributor:
+  name: Andrew Leap
+  link: [github.com/andyleap](http://github.com/andyleap)
+  external_resources:
+- '[Alpine Linux](http://www.alpinelinux.org/)'
+---
+
+*This is a Linode Community guide. Write for us and earn $250 per published guide.*
+<hr>
+
+[Alpine Linux](http://www.alpinelinux.org/) is a small, security oriented linux distro.
+It is regularly updated with security patches, and runs on the grsec kernel.  All binaries are staticly linked and built against musl libc.
+
+
+## Before You Begin
+
+1.  Familiarize yourself with Lish, as most of this guide will use Lish.
+
+2.  Understand that installing Alpine in this manner will destroy all existing data on the drives you install it on.  Backup all your data first.
+
+3.  The initial portion of this guide will involve creating the drives, so you may need to delete existing drives to have enough free space.
+
+## Setting up the Linode
+
+### Drives
+
+1.  Logon to the (Linode Manager)[https://manager.linode.com/linodes/] and select the linode you will be installing Alpine Linux on.
+
+2.  Create your boot drive, this should be 128~256mb, and the type should be ext4
+
+3.  Create your root drive, with as much space as you need, though if you want a swap partition, make sure you leave room for it.  This drive should be ext4 as well.
+
+4.  (Optionally) Create your swap partition with type swap. (512mb is a good amount of swap)
+
+### Configuration
+
+1. Create a new configuration for your Linode.  Alpine will need to be booted via the Grub 2 kernel, with all the helpers turned off, and boot, root, and swap assigned to /dev/sda, /dev/sdb, and /dev/sdc.
```

Comment:
"Configuration Profile", to match the official name.

Also, grub 2 isn't a kernel, despite the menu name, so might be clearer to say "Choose Grub 2 in the kernel menu, which will use grub 2 to boot Alpine's kernel from your disk image".

Listing disks that way, as A, B, C, to 1, 2, 3 is going to confuse some people. Might be worth just making it a bulleted list w/ the mappings.